src/vs/workbench/contrib/issue/electron-sandbox/issue.contribution.ts

```diff
@@ -51,6 +57,16 @@ class NativeIssueContribution extends BaseIssueContribution {
 			});
 		};
 
+		Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).registerConfiguration({
+			properties: {
+				'issueReporter.experimental.auxWindow': {
+					type: 'boolean',
+					default: productService.quality !== 'stable',
+					description: 'Enable the new experimental issue reporter in electron.',
+				},
+			}
+		});
```

Comment:
this gets registered properly, right? You can see the setting just fine and it doesn't have to be like... top level out of the file _next to_ the Contribution? The reason I ask is because since this `ctor` uses the `ConfigurationService` I'm not sure if it's too late and the configuration is already resolved.